dask/dataframe/tests/test_dataframe.py

```diff
@@ -1712,16 +1713,17 @@ def test_apply():
               df.apply(lambda xy: xy[0] + xy[1], axis='columns'))
 
     # inference
-    assert_eq(ddf.apply(lambda xy: xy[0] + xy[1], axis=1),
+    assert_eq(ddf.apply(lambda xy: xy[0] + xy[1], axis=1, meta=(None, int)),
               df.apply(lambda xy: xy[0] + xy[1], axis=1))
-    assert_eq(ddf.apply(lambda xy: xy, axis=1),
+    assert_eq(ddf.apply(lambda xy: xy, axis=1, meta={'x': int, 'y': int}),
               df.apply(lambda xy: xy, axis=1))
 
     # specify meta
     func = lambda x: pd.Series([x, x])
     assert_eq(ddf.x.apply(func, meta=[(0, int), (1, int)]), df.x.apply(func))
     # inference
-    assert_eq(ddf.x.apply(func), df.x.apply(func))
+    with pytest.warns(UserWarning):
```

Comment:
ISTM?

> Are we really expecting a warning in this kind of call?

`.apply` emits a UserWarning anytime `meta` isn't passed by the user.

> If the intent is to silence warnings without expecting them I'd rather have a specific context manager for that.

`pytest.warns(None)` will capture the warnings without making any assertions. I suppose that might be better for this test, as the intent is to check the behavior, not that a warning is or isn't raised. I'll change it here and in related tests.